docs/pages/reference/mesheryctl/subcommands/mesheryctl-app-view.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: mesheryctl app view
+permalink: reference/mesheryctl/apps/view
+type: reference
+display-title: "false"
+language: en
+command: app
+subcommand: view
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl app view [application name/id] [flags]
+
+<!-- Description of the command. Preferably a paragraph -->
+## Description
+
+{% assign name = site.data.mesheryctlcommands.cmds[page.command].subcommands[page.subcommand] %}
+{{ name.description }}
+
+<!-- Basic usage of the command -->
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl app view [application-name| application-id] | [flag]
+  mesheryctl app view BookInfo -o json
```

Comment:
you can remove it now as we are indicating here as `mesheryctl app view [application-name| application-id] [flag]` already, even it is in the `desc` section.